week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,11 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function validate(num) {
+  return (
+    typeof num === "number" && Number.isInteger(num) && num !== excludedNums
```

Comment:
`num !== excludedNums` is comparing a number to an array of numbers which will always be `true`. Did you check this gives the expected output?